iothub/device/src/PnPOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    internal class PnPOptions
```

Comment:
Recommend naming this PnpOptions. We should only be capitalizing the first letter of acronyms.